migrate_webhooks_to_v3/migrate_webhooks_to_v3.py

```diff
@@ -169,13 +231,21 @@ def main():
     ap.add_argument('-a', '--action', default='copy', choices=['copy',
                                                                'delete'],
                     help="Action to take on v1/v2 webhooks en masse")
+    ap.add_argument('-e',
```

Comment:
Tested this and only ran into a 401 when trying to use an API key associated with a US account and adding --europe / -e argument. On the other hand, EU key with no --europe / -e argument worked - no errors and webhook v3 copies created successfully.